web/server/entrypoint.go

```diff
@@ -57,7 +57,7 @@ func RunServer(ctx context.Context, args []string, _ golog.Logger) (err error) {
 	if argsParsed.Debug {
 		logConfig = golog.NewDebugLoggerConfig()
 	} else {
-		logConfig = golog.NewProductionLoggerConfig()
+		logConfig = golog.NewDevelopmentLoggerConfig()
```

Comment:
in addition to changing the timestamp format, this appears to set `Development` to true and change the EncodeLevel to `zapcore.CapitalColorLevelEncoder` from `zapcore.LowercaseLevelEncoder`.

are those safe/desirable changes?

Reply:
the former change i do not understand, but seems likely undesirable (setting `Development: true` to production builds).

the latter i'm fairly confident we do not want. colors are typically implemented via escape codes and will not render correctly in many places.

Reply:
fair, did a less invasive change - just updates the time encoder to ISO

Reply:
logs look like this now `2022-11-23T16:32:27.338-0500    info    robot_server    rpc/server.go:739       stopped cleanly`